kong/plugins/prometheus/exporter.lua

```diff
@@ -71,12 +75,43 @@ local function init()
                                          "Total bandwidth in bytes " ..
                                          "consumed per service/route in Kong",
                                          {"service", "route", "type"})
+
+
+
+  -- For all generic dbless node
+  if kong.configuration.database == "off" then
+    declarative = require("kong.db.declarative")
+
+    metrics.config_current_hash = prometheus:gauge("config_current_hash",
```

Comment:
Can we please name this and hybrid mode metric same?
The goal here should be to minimize the burden on the user to understand metrics.

Reply:
This is exposed not just for dataplane but also for kong dbless in tradition setup

Reply:
This metrics is now removed, as CP doesn't has this exposed anymore.